portfolio-project-matching-app/components/Button.module.css

```diff
@@ -0,0 +1,13 @@
+/* https://tailwindcss.com/docs/guides/nextjs */
+
+.btn {
+    @apply py-2 px-3 border-2 rounded-3xl focus:outline-none focus:ring-1;
+}
```

Comment:
Here, you might notice that `m-2` is missing. I was hoping we could leave the child components without any explicit margin requirements in favor of allowing the parent to set padding based on context.

Reply:
That makes sense to have the parent component responsible for margin.